matplotcheck/raster.py

```diff
@@ -57,14 +57,15 @@ def assert_colorbar_range(self, crange):
             ), "Colorbar maximum is not expected value:{0}".format(crange[1])
 
     def _which_label(self, label, all_label_options):
-        """Helper function for assert_legend_accuracy_classified_image
+        """Helper function for assert_legend_labels
         Returns string that represents a category label for label.
```

Comment:
This is a very confusing docstring! it seems like this helper actually tests to see which input labels (from the student plot) match the expected labels (from the instructor test). so let's clarify both the helper function name and the docstring. 

_check_labels or something like that? 

also the output of this method is odd. it returns None if the labels DO MATCH (i think??) and the label that is incorrect if not. My suggestion is that it returns a boolean that can be used to then pull out which provided labels from the student plot are incorrect.

here are my notes -- see what you think about this approach.
```
    labels_check = [_which_label(label, all_label_options[i]) for i, label
        in enumerate(labels)]

    # Check that each legend entry label is in one of all_label_options
    # So it might be useful here to atleast print out the label found that
    # does NOT match the expected label. if which_labels returns a boolean i
    # think you could get those values by doing labels[boolean array]. and
    # then print this out.
```